docsteady.py

```diff
@@ -2,5 +2,8 @@
 from docsteady import cli
 import sys
 
+import warnings
+warnings.filterwarnings("ignore", category=UserWarning, module='bs4')
```

Comment:
Can you add a comment here explaining why you need to do this? Also, in general we try to be consistent with quoting -- single vs double -- within a file. In theory turning on black would fix all of this.